build/ftest-base/pom.xml

```diff
@@ -14,7 +14,7 @@
 
   <properties>
     <version.shrinkwrap>1.2.6</version.shrinkwrap>
-    <version.shrinkwrap.resolver>2.0.0-beta-3</version.shrinkwrap.resolver>
+    <version.shrinkwrap.resolver>3.1.4</version.shrinkwrap.resolver>
```

Comment:
we should reuse the same property from root pom, and remove the above property in this pom.

Reply:
Though about this, too, but didn't dare to do it before. Is changed now.